src/components/Button/index.tsx

```diff
@@ -12,28 +12,28 @@ const buttonVariants = cva(
         primary:
           'bg-button-primary border-button-primary text-white ' +
           'hover:bg-button-primary-hover hover:border-button-primary-hover ' +
-          'focus:bg-button-primary-focus focus:border-button-primary-focus ' +
-          'disabled:bg-button-disabled disabled:text-foreground-subtle disabled:border-button-disabled',
+          'focus:bg-button-primary-focus focus:border-button-primary-focus focus:shadow-blue ' +
+          'disabled:bg-button-primary-disabled disabled:text-foreground-subtle disabled:border-button-primary-disabled ',
         secondary:
           'bg-white text-grey-900 border-border ' +
           'hover:bg-button-secondary-hover hover:border-button-secondary-border-hover ' +
-          'focus:bg-button-secondary-focus focus:border-button-secondary-border-focus ' +
+          'focus:border-button-secondary-border-focus focus:shadow-blue ' +
```

Comment:
While we're here, let's convert this file from concatenated strings to arrays:
```tsx
[
...
  secondary:
          ['bg-white text-grey-900 border-border ' ],
          ['hover:bg-button-secondary-hover hover:border-button-secondary-border-hover ']
...
]
```
You can see I've done something similar with the Checkbox :D